### ISSUE TITLE
🐛: Uploading files using web form causes error

### ISSUE THREAD

**Issue by bilalshaikh42** [reproducible-biomedical-modeling/CRBM-Viz#3719]
### Location

runBiosimulations

### What happened?

Using the webform to upload files, an error is presented when trying to create the formdata for submission. 
Almost certainly due to 0ae13a8b7b755988c016d8985d775b45a82c5844 having a different type for the uploaded files

### What made it happen?

Submit a file for simulations

### Version

Development (Deployed on a .dev domain)

### Relevant log output

```shell
TypeError: Failed to execute 'append' on 'FormData': parameter 2 is not of type 'Blob'.
    at p.submitJobForFile (dispatch.service.ts:155)
    at i.onFormSubmit (dispatch.component.ts:760)
    at dispatch.component.html:5
    at Wl (core.mjs:15291)
    at Object.c [as next] (core.mjs:15329)
    at Object.next (Subscriber.js:110)
    at Z._next (Subscriber.js:60)
    at Z.next (Subscriber.js:31)
    at Subject.js:31
    at l (errorContext.js:19)
handleError @ error-handler.ts:36
n4 @ core.mjs:10976
Wl @ core.mjs:15294
c @ core.mjs:15329
(anonymous) @ Subscriber.js:110
_next @ Subscriber.js:60
next @ Subscriber.js:31
(anonymous) @ Subject.js:31
l @ errorContext.js:19
next @ Subject.js:26
emit @ core.mjs:22776
onSubmit @ forms.mjs:5352
(anonymous) @ forms.mjs:5434
Wl @ core.mjs:15291
c @ core.mjs:15329
(anonymous) @ platform-browser.mjs:546
invokeTask @ zone.js:406
Se.onInvokeTask @ task-tracking.js:65
invokeTask @ zone.js:405
onInvokeTask @ core.mjs:25864
invokeTask @ zone.js:405
runTask @ zone.js:178
invokeTask @ zone.js:487
j @ zone.js:1600
x @ zone.js:1626
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jonrkarr** [comment]
Why did this need to be changed? Because we don't have unit tests, changes like this have to be checked manually. Unless there's a compelling reason, we have other things to focus on.

**Comment by bilalshaikh42** [comment]
This was a part of the changes in #3668. I noticed it when submitting projects from the physiome repo.  I have pushed a fix in #3721. Ready to merge with a review.
